ios/Helio.xcodeproj/project.pbxproj

```diff
@@ -1008,7 +1008,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export BUNDLE_COMMAND=\"ram-bundle\"\nexport NODE_BINARY=node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
+			shellScript = "export BUNDLE_COMMAND=\"ram-bundle\"\nexport NODE_BINARY=/Users/agitafirstawan/.nvm/versions/node/v16.14.2/bin/node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded path to the node binary could cause issues if the project is built on a different machine where Node.js is installed in a different location or if the user does not have nvm installed. Consider using `env` to find the node binary in the PATH.
```diff
- export NODE_BINARY=/Users/agitafirstawan/.nvm/versions/node/v16.14.2/bin/node
+ export NODE_BINARY=$(which node)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->